src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -260,17 +260,6 @@ file that was distributed with this source code.
                           $('#{{ id }}_hidden_inputs_wrap').html('<input type="hidden" name="{{ full_name }}" value="'+xhr.responseJSON.objectId+'" />');
                         {% endif %}
 
-                        // append to Select2
-                        autocompleteInput.attr('data', data).append(data).trigger('change');
-
-                        // manually trigger the `select2:select` event
-                        autocompleteInput.attr('data', data).trigger({
-                            type: 'select2:select',
-                            params: {
-                                data: data
-                            }
-                        });
-
```

Comment:
There was a comment to remove this code, but I'm not sure about this.

@cecicifu Can you tell me what to do with this code on master ?

Reply:
Just remove attr function from lines 264 and 267.
264: `autocompleteInput.append(data).trigger('change');`
267: `autocompleteInput.trigger({`

Reply:
Should be ok

Reply:
I approved these changes, but i cannot review the others